Measurements/Measurements_Workbook.ipynb

```diff
@@ -0,0 +1,453 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Measurement Kata Workbook\n",
+    "\n",
+    "**What is this workbook?** \n",
+    "\n",
+    "A workbook is a collection of problems, accompanied by solutions to them. The explanations focus on the logical steps required to solve a problem; they illustrate the concepts that need to be applied to come up with a solution to the problem, explaining the mathematical steps required.\n",
+    "Note that a workbook should not be the primary source of knowledge on the subject matter; it assumes that you've already read a tutorial or a textbook and that you are now seeking to improve your problem-solving skills. You should attempt solving the tasks of the respective kata first and turn to the workbook only if stuck. While a textbook emphasizes knowledge acquisition, a workbook emphasizes skill acquisition.\n",
+    "This workbook describes the solutions to the problems offered in the [Measurement Kata](https://notebooks.gesis.org/binder/jupyter/user/microsoft-quantumkatas-iacsymjt/notebooks/Measurements/Measurements.ipynb). Since the tasks are offered as programming problems, the explanations also cover some elements of Q# that might be non-obvious for a first-time user.\n",
+    "\n",
+    "**What you should know for this workbook?**\n",
+    "\n",
+    "You should be familiar with the following concepts before tackling the Measurements kata (and this workbook):\n",
+    "\n",
+    "1.\tBasic linear algebra\n",
+    "2.\tThe concept of qubit and multi-qubit systems\n",
+    "3.\tSingle-qubit and multi-qubit quantum gates\n",
+    "4.\tSuperposition\n",
+    "\n",
+    "To begin, first prepare this notebook for execution (if you skip the first step, you'll get \"Syntax does not match any known patterns\" error when you try to execute Q# code in the next cells; if you skip the second step, you'll get \"Invalid test name\" error):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%package Microsoft.Quantum.Katas::0.10.1911.1607"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%workspace reload"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part I. Discriminating Orthogonal States"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Task 1.1.  $|0\\rangle$ or $|1\\rangle$ ?\n",
+    "\n",
+    "**Input:** A qubit which is guaranteed to be in either the $|0\\rangle$ or the $|1\\rangle$ state.\n",
```

Comment:
Please review the workbook to check that the task description renders in same way as in the original kata. Looks like some line breaks were lost, so input, output and comments are all in the same paragraph:
![image](https://user-images.githubusercontent.com/10113024/73402815-f1f93e00-42a2-11ea-958c-55acc64ae72d.png)
In this task all these sections are small, but later tasks have longer task descriptions, and around task 1.6 they become really hard to read.